joomla/joomla-cms
# [4.0] Use Custom Element tabs on Global Config

### Fixes
#25876 

### Summary of Changes

Replace the Bootstrap tabs on the Global Config page with Custom Elements.

### Testing Instructions

1. Go to `administrator/index.php?option=com_config`
2. Navigate through the tabs and ensure they display the correct content
